indexing.py

```diff
@@ -0,0 +1,20 @@
+# Indexing Operator []
+
+# you are going to learn Indexing now
+
+x=str(input("enter a string value"))
+print(x)
+print("you entered", x)
+if len(x) % 2 ==0:
+ #print ("your string is even")
```

Comment:
This isn't working as expected for strings with length as even number.

<img width="503" alt="image" src="https://user-images.githubusercontent.com/38807024/200735378-01fa1643-11a4-42dd-b35e-2ccb29266d15.png">

Reply:
Middle characters in this scenario should be "t" and "h". please correct your code.